GreenField-Org/ultracrewmobile
# #207 updating expo version and dependencies

closes #207 

### Steps To Test 

- run ```npm install```
- run ```npx expo start```
- Use your camera on your phone to launch Expo Go 
- Verify the application still loads and runs as expected. 
- in your terminal type ```w``` for web
- verify the app loads in the web browser and runs as expected.
